### PR TITLE
inspector light gizmo

### PR DESCRIPTION
**Pull request by TrevorDev**

### CODE REVIEW

File: inspector/src/components/sceneExplorer/entities/sceneTreeItemComponent.tsx [hunk 1]
```diff
@@ -10,26 +10,29 @@ import { faSyncAlt, faImage, faCrosshairs, faArrowsAlt, faCompress, faRedoAlt }
...
     private _selectedEntity: any;
 
     constructor(props: ISceneTreeItemComponentProps) {
         super(props);
-
-        const scene = this.props.scene;
         let gizmoMode = 0;
-        if (scene.reservedDataStore && scene.reservedDataStore.gizmoManager) {
-            const manager: GizmoManager = scene.reservedDataStore.gizmoManager;
+        if (props.globalState.gizmoManager) {
```
Comment (deltakosh) [inline inspector/src/components/sceneExplorer/entities/sceneTreeItemComponent.tsx]: Why do you need to store it globally and not on the scene?

File: inspector/src/inspector.ts [hunk 2]
```diff
@@ -404,6 +404,12 @@ export class Inspector {
                 this._CreateActionTabs(scene, options, parentControl);
             }
         }
+
+        // Light gizmos
+        scene.lights.forEach((l)=>{
+            debugger;
```
Comment (deltakosh) [inline inspector/src/inspector.ts]: Debugger?

File: inspector/src/components/sceneExplorer/entities/sceneTreeItemComponent.tsx (hunk 1 above)
Comment (TrevorDev) [reply inspector/src/components/sceneExplorer/entities/sceneTreeItemComponent.tsx]: At one point I didn't have a reference to scene so I moved it to global to access it. I think all places its used now have access to the scene but I left it thinking it was better to add to globalstate than scene.reservedDataStore . I can revert if you prefer the other way.

Comment (deltakosh) [reply inspector/src/components/sceneExplorer/entities/sceneTreeItemComponent.tsx]: Yes please, so it is more unified

**Pull request merged**